seed/seedwriter/seed20.go

```diff
@@ -159,7 +159,10 @@ func (tr *tree20) mkFixedDirs() error {
 		return err
 	}
 
-	return os.MkdirAll(tr.systemDir, 0755)
+	if err := os.MkdirAll(filepath.Dir(tr.systemDir), 0755); err != nil {
+		return err
+	}
+	return os.Mkdir(tr.systemDir, 0755)
```

Comment:
what will we have to do if hit this error at runtime?  delete the directory tree?  do we know enough from this to distinguish it?

Reply:
There's a bit more context needed which is not available at this level. However, when creating a recovery system in devicestate, we can make some choices about further course of action.

Reply:
I still think we should turn the directory exists error from here into something specific like a ErrSystemAlreadyExists

Reply:
This is done now.